src/services/email.js

```
@@ -0,0 +1,46 @@
+// src/services/email.js
+const nodemailer = require('nodemailer');
+
+const NO_EMAIL = process.env.NO_EMAIL === '1';
+
+let transporter = null;
```

`transporter` is initialized to null and only set when NO_EMAIL is false. If transporter creation fails or returns null this code will later call `transporter.sendMail` and crash. Validate that `transporter` exists before using it (or fall back to logging) and surface a clear error. Consider wrapping transport creation in try/catch and failing fast with a helpful log.